src/os/bluestore/BlueFS.cc

```diff
@@ -378,7 +378,6 @@ int BlueFS::_write_super()
   ::encode(crc, bl);
   assert(bl.length() <= get_super_length());
   bl.append_zero(get_super_length() - bl.length());
-  bl.rebuild();
```

Comment:
but `bl.rebuild()` is not identical to `bl.rebuild_aligned()` used by `BlockDevice::aio_write()`. the former create a new memory chunk, and concatenate all memory segments into the newly allocated one, while in the case of `KernelDevice`, the later aligns unaligned buffer segments. and in the case of NVMEDevice, it does not rebuild the buffer segments at all, instead, it splits the buffer to be written into slices of `data_buffer_size` bytes before sending them down to underlying storage.